octobot_trading/data/portfolio.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, exchange_name, is_simulated=False):
 
         self.logger = get_logger(f"{self.__class__.__name__}{'Simulator' if is_simulated else ''}[{exchange_name}]")
         self.lock = Lock()
-        self.portfolio = None
+        self.portfolio = {}
```

Comment:
?

Reply:
A portfolio should be initialize and will be set to `{}` during initialize()

Reply:
Using None is producing a lot of exceptions at different places when portfolio is never initialized (when exchange creds are wrong / missing for ex)

Reply:
I don't thing having a None default value is adding any value to this

Reply:
It doesn't make sense to create a portfolio and not initialize it.